spyder/plugins/completion/languageserver/tests/conftest.py

```diff
@@ -39,7 +39,7 @@ def __init__(self):
         self.lsp_editor_settings = {}
 
     @Slot(dict, str)
-    def register_completion_server_settings(self, settings, language):
+    def register_completion_settings(self, settings, language):
         self.lsp_editor_settings[language] = settings
```

Comment:
Maybe since the signature name was changed we could add a docstring describing the arguments? (probably `settings` is the dict with the config that follows some specific structure, right?)

Reply:
You're right. I'll add docstrings for this and the other methods I changed in my refactoring.